doc/md_src_files/040_stakeholders-scope.md

```diff
@@ -116,16 +116,21 @@ Table 2:<a name="table-supplier-outcomes"></a> Proposed Meaning for Supplier Pri
+Mitigation techniques may not affect the SSVC decision. The mitigating action may not be permanent or work as designed. For example, the implementation of a firewall or IDS rule to mitigate [*System Exposure*](#system-exposure) from open to controlled is only valid until someone changes the rule. In the area of Financial impacts, the caps on the insurance may be too low to act as a mitigation. The Physical impact may be increased by incorrect installation of the physical barriers designed to restrict a robot’s ability to interact with humans. The Mission impact could be increased when a disaster recovery test-run identifies problems with an alternate business flow.
 
 A mitigation that successfully changes the value of a decision point may shift the priority of further action to a reduced state. An effective firewall or IDS rule coupled with an adequate change control process for rules may be enough to reduce the priority where no further action is necessary. In the area of Financial impacts, a better insurance policy may be purchased, providing necessary fraud insurance. Physicial well-being impact may be reduced by testing the physicial barriers designed to restrict a robot's ability to interact with humans. Mission impact could be reduced by correcting the problems identified in a disaster recover test-run of the alternate business flow. If applying a mitigation reduces the priority to **defer**, the deployer may not need to apply a remediation if it later becomes available. [Table 3](#table-deployer-outcomes) displays the action priorities for the deployer, which are similar to the supplier case.
 
-<!--**Talk about applying other mitigations here** TODO
--->
+As opposed to mitigation, applying a remediation finishes an SSVC analysis of a deployed system.
+While specific vulnerabilities in specific systems can be remediated, the vulnerability cannot be 'disposed of' or eliminated from future consideration within an IT environment.
+Since software and systems are dynamic, a single vulnerability can be re-introduced after initial remediation through updates, software rollbacks, or other systemic actions that change the operating conditions within an environment.
+It is therefore important to continually monitor remediated systems for re-introduced vulnerabilities even after initial remediation.
```

Comment:
Should this chunk be talking about both remediated and newly deployed systems? I think you've covered the case where

1. deployed system is remediated
2. vul is reintroduced to deployed system

but not 
1. vul is remediated in deployed systems
2. new system deployed with vul

Reply:
https://github.com/CERTCC/SSVC/pull/128/commits/4b2d110e101a9c6e0138c1fc1066e8a2316626c7

Reply:
Consider deleting this part of the sentence:

'disposed of' or

Reply:
regarding the "new system deployed with vul"  

Deploying a new system with the same vul should be detected by other mitigations.  (should) continuous scanning; auto patch deployment; configuration management -- one of these should identify the new system with the same (now old) vulnerability that was remediated.   Adding a new system puts us at the beginning of the decision tree once the vul is identified. Adding a new system with the same vul shouldn't happen, but it does.